model_analyzer/config/input/config_command_profile.py

```diff
@@ -312,33 +303,33 @@ def objective_list_output_mapper(objectives):
         constraints_scheme = ConfigObject(
             schema={
                 'perf_throughput':
-                ConfigObject(schema={
-                    'min': ConfigPrimitive(int),
-                }),
+                    ConfigObject(schema={
```

Comment:
Is this PR formatted using format.py?

Reply:
Guan, created a script to autoformat using format.py in a git commit. https://github.com/triton-inference-server/common/blob/main/tools/pre-commit